experiments/cifar10/dataset.py

```diff
@@ -1,12 +1,19 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
 import numpy as np
-import cPickle
 import os
-from skimage import img_as_float
+
 from functools import partial
+from six.moves import cPickle
+from skimage import img_as_float
+
 
 def load_cifar10_batch(fpath, one_hot=True, as_float=True):
     with open(fpath, 'rb') as f:
-        data = cPickle.load(f)
+        data = cPickle.load(f, encoding='latin1')
```

Comment:
?

Reply:
Many stacks overflowed here. Just let it be.